tile-generation/src/main/scala/com/oculusinfo/tilegen/datasets/TilingTaskParameters.scala

```diff
@@ -66,7 +66,8 @@ case class TilingTaskParameters (name: String,
 																 minimumSegmentLength: Option[Int] = None,
 																 maximumSegmentLength: Option[Int] = None,
 																 maximumLeaderLength: Option[Int] = None,
-																 lineType: LineDrawingType = LineDrawingType.Lines)
+																 lineType: LineDrawingType = LineDrawingType.Lines,
+																 filterToRegion: Boolean = false)
```

Comment:
Can we move this up before the line parameters?

We should probably separate the line parameters out, and have separate line and point configurations, but that's part of a larger, different task; for the moment, keeping all the point stuff first (so the line parameters can just be defaulted when not in use) makes use a lot easier.